pkg/controller/cloudlet_api.go

```diff
@@ -2079,6 +2092,19 @@ func (s *CloudletApi) UsesResTagTable(key *edgeproto.ResTagTableKey) *edgeproto.
 	return nil
 }
 
+func (s *CloudletApi) UsesZone(zoneKey *edgeproto.ZoneKey) []string {
```

Comment:
I would change this to `cloudletsInZone`, or something, otherwise this sounds like it should return a bool.

Reply:
Changed to CloudletsUsingZone()

Reply:
Thanks!